geostore/tasks.py

```diff
@@ -22,3 +28,23 @@ def layer_relations_set_destinations(relation_id):
         feature_update_relations_destinations.delay(feature_id, relation_id)
 
     return True
+
+
+@shared_task
+def generate_shapefile_async(layer, user):
```

Comment:
il ne faut JAMAIS passer d'instance en parametre de function asynchrone. toujours des données serializables. ces parametres vont etre serializés en json par celery pour etre stocké dans le broker. çà ne marchera pas en prod avec un worker.

```suggestion
def generate_shapefile_async(layer_id, user_id):
```

Reply:
ok